app/error/userInactiveController.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+(function() {
+    var app = angular.module("app");
+    
+    app.controller("UserInactiveController", function UserInactiveController(AuthService) {
+        var userInactiveCtrl = this;
+
+        Object.defineProperty(userInactiveCtrl, 'user', {
+            get: function() {
+                return AuthService.user;
+            }
+        });
+
+        /*
+        * To implement this function when search of institution finished
```

Comment:
Coloca aqui um TODO.
Nesse estilo: com seu nome e data
 /*TODO: Change to AuthService.getCurrentUser()
                      @author: Mayza Nunes 14/07/2017
   */